### DISCUSSION TITLE
Could you make a version of CHDMAN for Android?

### DISCUSSION THREAD

**Discussion by youtubeeicarlos** [mamedev/discussions#71] [Q&A]
I'm just asking because from what I heard, you were the geniuses who created CHD for MAME, I would like to know if you could create an apk that does the work of CHDMAN on Android, to this day there is nothing that does such a thing.  Android is scarce in resources, there is no apk that extracts and compresses in ISO (not one that is truly functional and without defects), there is no apk that compresses and decompresses in CHD format and CSO format, it is complicated to depend on PC for everything, even more so for those who use MAME through Android's Retroarch.

**Comment by rb6502** [comment]
We don't officially support MAME itself on Android (MAME4Droid was recently revived, but that's not us), and we definitely don't support RetroArch.  I don't know that CHDMAN would run well on Android anyway, as it wants lots of CPU and RAM (compressing is much heavier than decompressing).
